xapian-applications/omega/index_file.cc

```diff
@@ -531,8 +531,7 @@ index_add_document(const string & urlterm, time_t last_altered,
 
 void
 index_mimetype(const string & file, const string & urlterm, const string & url,
-	       const string & ext,
-	       const string &mimetype, DirectoryIterator &d,
+	       const string & ext, string &mimetype, DirectoryIterator &d,
```

Comment:
This change is obscure, and it'd be clearer what you've done if you preserved the line breaks where they were. (As you did in the header file.)

Reply:
I will try to avoid this kind of changes.